profile_bluesky/startup/instrument/devices/aps_source.py

```diff
@@ -13,53 +13,51 @@
 
 import apstools.devices
 from ..framework import sd
-from ophyd import Device,Component, Signal
+from ophyd import Device, Component, Signal, EpicsSignal
 
 aps = apstools.devices.ApsMachineParametersDevice(name="aps")
 sd.baseline.append(aps)
 
+
 class MyUndulator(apstools.devices.ApsUndulator):
```

Comment:
Some of these changes would be good to submit in [apstools](https://github.com/BCDA-APS/apstools).  Care to submit an issue (or PR)?

Reply:
Looking at this again, should I make tracking just a signal (instead of having a _tracking)? I can move the stuff that is currently under @tracking.setter to Signal.put.

Reply:
My rule of thumb: Make it a Signal if you want to include it with `.read()` or in the configuration for data acquisition.  Make it an attribute (with setter & getter) if you want it to be a user control for configuration or operation (and not remembered in some way).

Tracking might be something you'd want to know later whether or not you had it set.  So, `Signal`.

Reply:
Ok. I will make it a signal as in:
https://github.com/APS-4ID-POLAR/ipython-polar/blob/63e2376ab22bc8db1fd9435943892d2cd5779382/profile_bluesky/startup/instrument/devices/phaseplates.py#L26